flask_caching/contrib/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .ignitecache import IgniteCache
+
+__all__ = ['IgniteCache']
```

Comment:
I'd like to only use the full important path for contributed caches -- without importing the cache class here.